internal/plugins/golang/v2/api.go

```diff
@@ -63,23 +54,5 @@ func (p *createAPIPlugin) Run() error {
 
 // SDK plugin-specific scaffolds.
 func (p *createAPIPlugin) run() error {
-
```

Comment:
this code was centralized in internal/plugins/sdk since is required for the other plugins as well.